### PR TITLE
Lighthouse preset fix

### PR DESCRIPTION
**Pull request by timothy-ch-cheung**
Switch to use the recommended lighthouse config, fix findings to get a passing report

### CODE REVIEW

File: src/pages/landing/index.tsx [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
 import { styled } from "linaria/react"
-import FeelingLogo from "../../assets/icons/logo.svg"
-import TwitterSvg from "../../assets/icons/twitter.svg"
-import EmailSvg from "../../assets/icons/email.svg"
+
+import FeelingLogo from "../../assets/icons/logo.svg?sprite"
+import TwitterSvg from "../../assets/icons/twitter.svg?sprite"
+import EmailSvg from "../../assets/icons/email.svg?sprite"
+// @ts-ignore
+import IPhone from "../../../public/iphone_crop.png?resize&sizes[]=300&sizes[]=500&sizes[]=850"
```
Comment (timothy-ch-cheung) [inline src/pages/landing/index.tsx:8]: Not sure how to make typescript recognise this type - any suggestions?

**Comment by timothy-ch-cheung** [comment]
Ready for review:
CodeCov flagging the index file (Hello world) which I don't think is necessary to test.

Still one remaining lighthouse failure ("unused-js" - https://web.dev/unused-javascript/), not sure how to fix (checked lighthouse coverage and its not clear which lines its classing as unused), just upped the threshold to 4 for now. 

Lighthouse flagging a warning for not using next-gen formats - can't fix this with next-optimized-images yet, seems to be a bug that someone else has also seen (https://github.com/cyrilwanner/next-optimized-images/issues/207)

File: package.json [hunk 2]
```diff
@@ -21,18 +21,22 @@
     ]
   },
   "dependencies": {
+    "@types/imagemin-svgo": "7.0.0",
+    "@types/svg-sprite-loader": "3.9.2",
```
Comment (pavsidhu) [inline package.json:25]: Should add these to dev deps instead, same in the package-lock.json

File: src/pages/_document.tsx [hunk 3]
```diff
@@ -38,6 +38,12 @@ export default class MyDocument extends Document {
             href="/icons/safari-pinned-tab.svg"
             color="#caeaf8"
           />
+          <link rel="manifest" href="/manifest.json" />
+          <meta name="theme-color" content="#fefefe" />
```
Comment (pavsidhu) [inline src/pages/_document.tsx:42]: Not 100% sure we need this as it's already specified in public/manifest.json should probably double check this

File: src/pages/landing/index.tsx (hunk 1 above)
Comment (pavsidhu) [reply src/pages/landing/index.tsx:8]: Seems like a common issue: https://github.com/cyrilwanner/next-optimized-images/issues/103
You think maybe in the type declaration if we did `*.png*` would that work?

**Review by pavsidhu — CHANGES_REQUESTED** [review]

File: src/pages/_document.tsx (hunk 3 above)
Comment (timothy-ch-cheung) [reply src/pages/_document.tsx]: We do need the tag (see: https://github.com/feelingapp/feeling-app/runs/1168465935?check_suite_focus=true) - I've changed it to match the one in manifest

File: package.json (hunk 2 above)
Comment (timothy-ch-cheung) [reply package.json:25]: These dependencies weren't needed (I added them to try to fix the typing of next-optimized-images but forgot to remove them)

**Review by pavsidhu — APPROVED** [review]

**Pull request merged**